ngc2oci

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+#
+# ngc2oci - Simple shell tool to pull from Nvidia Cloud and create an OCI image.
+# Copyright (C) 2018  Vanessa Sochat
```

Comment:
and if you would like this to have the same header I'm OK with that too, I am not sure how to do this properly since I created a new file! +1 CONTRIBUTORS.md file, although it might also make sense to have a note here (as in the blob2oci)

Reply:
I agree, the header is fine as you did it, also the note. I'll just add `CONTRIBUTORS.md` in addition in the future as the "central collection point" of all contributions ;-).